plugins/callback/diy.py

```diff
@@ -0,0 +1,1421 @@
+    runner_on_start_msg_color:
+      description:
+        - Output color to be used for runner_on_start_msg
+        - Template should render a L(valid color value,#notes)
+      ini:
+        - section: callback_diy
+          key: runner_on_start_msg_color
+      env:
+        - name: ANSIBLE_CALLBACK_DIY_RUNNER_ON_START_MSG_COLOR
+      vars:
+        - name: ansible_callback_diy_runner_on_start_msg_color
+      required: False
+      type: string
+
+    runner_on_no_hosts_msg:
+      description: Output to be used for callback runner_on_no_hosts
+      ini:
+        - section: callback_diy
+          key: runner_on_no_hosts_msg
+      env:
+        - name: ANSIBLE_CALLBACK_DIY_RUNNER_ON_NO_HOSTS_MSG
+      vars:
+        - name: ansible_callback_diy_runner_on_no_hosts_msg
+      required: False
+      type: string
+
+    runner_on_no_hosts_msg_color:
+      description:
+        - Output color to be used for runner_on_no_hosts_msg
+        - Template should render a L(valid color value,#notes)
+      ini:
+        - section: callback_diy
+          key: runner_on_no_hosts_msg_color
+      env:
+        - name: ANSIBLE_CALLBACK_DIY_RUNNER_ON_NO_HOSTS_MSG_COLOR
+      vars:
+        - name: ansible_callback_diy_runner_on_no_hosts_msg_color
+      required: False
+      type: string
+
+    playbook_on_setup_msg:
+      description: Output to be used for callback playbook_on_setup
+      ini:
+        - section: callback_diy
+          key: playbook_on_setup_msg
+      env:
+        - name: ANSIBLE_CALLBACK_DIY_PLAYBOOK_ON_SETUP_MSG
+      vars:
+        - name: ansible_callback_diy_playbook_on_setup_msg
+      required: False
+      type: string
+
+    playbook_on_setup_msg_color:
+      description:
+        - Output color to be used for playbook_on_setup_msg
+        - Template should render a L(valid color value,#notes)
+      ini:
+        - section: callback_diy
+          key: playbook_on_setup_msg_color
+      env:
+        - name: ANSIBLE_CALLBACK_DIY_PLAYBOOK_ON_SETUP_MSG_COLOR
+      vars:
+        - name: ansible_callback_diy_playbook_on_setup_msg_color
+      required: False
+      type: string
+'''
+
+EXAMPLES = r'''
+ansible.cfg: >
+  # Enable plugin
+  [defaults]
+  stdout_callback=diy
+
+  [callback_diy]
+  # Output when playbook starts
+  playbook_on_start_msg="diy output(via ansible.cfg): playbook example: {{ ansible_callback_diy.playbook.file_name }}"
+  playbook_on_start_msg_color=yellow
+
+  # Comment out to allow default plugin output
+  # playbook_on_play_start_msg="PLAY: starting play {{ ansible_callback_diy.play.name }}"
+
+  # Accept on_skipped_msg or ansible_callback_diy_runner_on_skipped_msg as input vars
+  # If neither are supplied, omit the option
+  runner_on_skipped_msg="{{ on_skipped_msg | default(ansible_callback_diy_runner_on_skipped_msg) | default(omit) }}"
+
+  # Newline after every callback
+  # on_any_msg='{{ " " | join("\n") }}'
+
+playbook.yml: >
+  ---
+  - name: "default plugin output: play example"
```

Comment:
```suggestion
  - name: "Default plugin output: play example"
```
https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#examples-block